decentraland/unity-renderer
# feat: skybox satellite planar horizon

Counterpart PR: https://github.com/decentraland/unity-renderer/pull/2273

Three new features for procedural skybox:

Horizon Plane: New horizon plane is integrated with ground disc.
Satellite layer: Any prefab can be used as a satellite and it will render on the secondary camera.
Planar layer: This layer are specific for particle system prefabs. particle system should be of type circle or hemisphere only.


To test this PR: 
- Play around the world, the skybox configuration should be similar to the one in production
- There's a config reverted (68bb60128c23f1957d52cbd70bcbd7b43cbbed4d) with satellites and such, you can take it to test the skybox editor.
- Test the skybox editor `Window/Skybox Editor`. Best place to test it is in the `Test Scene` but bear in mind that playmode is not working properly in that scene, if you want to change the skybox in realtime (while playing) use the InitialScene.
